components/inputs/input-text-mixin.js

```diff
@@ -0,0 +1,31 @@
+export const InputTextMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			ariaInvalid: { type: String, attribute: 'aria-invalid' },
+			ariaLabel: { type: String, attribute: 'aria-label' },
```

Comment:
I removed `aria-labelledby` from the behavior/mixin because since it takes an ID it won't actually work across shadow boundaries anyway.